src/patterns/definitions.owl

```diff
@@ -63759,8 +63803,8 @@ EquivalentClasses(<http://purl.obolibrary.org/obo/OBA_2040789> ObjectIntersectio
 
 # Class: <http://purl.obolibrary.org/obo/OBA_2040790> (level of dehydrogenase/reductase SDR family member 6 in blood serum)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute_location"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/OBA_2040790> "The amount of a dehydrogenase/reductase SDR family member 6 when measured in blood serum."^^xsd:string)
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute_location"^^xsd:string) <http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/OBA_2040790> "blood serum dehydrogenase/reductase SDR family member 6 amount"^^xsd:string)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute_location"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/OBA_2040790> "The amount of a 3-hydroxybutyrate dehydrogenase type 2 when measured in blood serum."^^xsd:string)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute_location"^^xsd:string) <http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/OBA_2040790> "blood serum 3-hydroxybutyrate dehydrogenase type 2 amount"^^xsd:string)
```

Comment:
The label change is pretty significant - is this intended?

Reply:
Good catch! I think I reused an ID that I had already assigned in another PR.

Reply:
I will double check the OBA IDs.